Swat/SwatGroupedFlydown.php

```diff
@@ -23,15 +21,12 @@ class SwatGroupedFlydown extends SwatTreeFlydown
      *
      * @throws SwatException if the tree more than 3 levels deep
      */
-    public function setTree(SwatTreeFlydownNode $tree)
+    public function setTree($tree): void
     {
         $this->checkTree($tree);
```

Comment:
This should be adjusted to convert a data tree to a treeflydown node before calling checkTree. Then the interface can have a strong type for both SwatGroupedFlydown and SwatTreeFlydown. Both could receive a datatree.